tests/commands_test.rs

```diff
@@ -67,6 +67,9 @@ fn save_can_write_out_csv() {
     })
 }
 
+// This test is more tricky since we are checking for binary output. The output rendered in ASCII is (roughly):
+// �authors+0Yehuda Katz <wycats@gmail.com>descriptionA shell for the GitHub eraedition2018licenseISCnamenuversion0.2.0
```

Comment:
hmm, we may want to elide this line so people don't report it as a bug later :)

Reply:
Oh, feel free to remove this :)